_common/core_ref.md

```diff
@@ -1,5 +1,18 @@
 ---
-layout: externalurl
-redirect_url: https://core.blockstack.org/
+layout: core
 title: "Stacks Node API"
```

Comment:
I fear this title may be a source of confusion. It can be read as having two meanings:
- "Stacks Node" API
- "Stacks" Node API (as in, a Stacks NodeJS API)

Maybe title it: `stacks-node` API, or Stacks Blockchain API?

Reply:
Good point. I like `Stacks Blockchain API`, could cover both sidecar and RPC